packages/react-data/src/fetch/ReactDataFetch.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Wakeable} from 'shared/ReactTypes';
+
+import {readCache} from 'react/unstable-cache';
+
+const Pending = 0;
+const Resolved = 1;
```

Comment:
Probably `Fulfilled` would be a more correct word here

https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md#states
https://jakearchibald.com/2014/resolve-not-opposite-of-reject/

Reply:
Also I found other usages of `Resolved` variable name in ReactCache, and ReactLazy. It seems that renaming won't be a breaking change. Let me know if I can help with this.

Reply:
Makes sense. We can rename this later, doesn't really matter how it's called internally.

Reply:
The term Fulfilled makes sense in combination with the term Promise since you "fulfill a promise to someone".

But these are not Promises. So not sure we need to stick to the same terminology. Maybe we should go with something completely different to make that clear.

Reply:
@seangransee So, fallowing the same reasoning it can be called,`Completed` or `Finished`.

Reply:
Or `Done`